AuthenticationSimpleSample/README.md

```diff
@@ -0,0 +1,46 @@
+---
+name: .NET MAUI - Authentication Azure AD B2C
+description: "This sample demonstrates how to implement authentication using Azure AD B2C (using external platforms)"
+page_type: sample
+languages:
+- csharp
+- xaml
+products:
+- dotnet-maui
+- dotnet-core
+urlFragment: authentication-B2C
+---
+<h1 align="center">Simple Azure AD B2C Authentication Sample</h1>
+
+<div align="center">
+This sample demonstrates how to implement authentication using Azure AD B2C to login with Microsoft and other external platforms. </br>
+This sample currently works on Windows, Android and iOS </br></br>
+</div>
+
+## Login Page
+<p align="center">
+    <img width="450" src=./Screenshots/login.png>
+</p>
+
+## Azure Login Page
+This page is prompted after clicking the login button
+<p align="center">
+    <img width="450" src=./Screenshots/login2.png>
+</p>
+
+## Setting Page
+<p align="center">
+    <img width="450" src=./Screenshots/settingsPage.png>
+</p>
+
+Before running the app:
+- [Create project and deploy the backend](https://docs.microsoft.com/en-us/azure/developer/mobile-apps/azure-mobile-apps/quickstarts/maui/#deploy-the-backend-to-azure)
+- [Register the application](https://docs.microsoft.com/en-us/azure/developer/mobile-apps/azure-mobile-apps/quickstarts/maui/#deploy-the-backend-to-azure)
```

Comment:
We like to remove the locales when sharing URLs, so that they auto-populate depending on people's resident country!

Make sure to delete the locales from all the Microsoft docs

```suggestion
- [Create project and deploy the backend](https://docs.microsoft.com/azure/developer/mobile-apps/azure-mobile-apps/quickstarts/maui/#deploy-the-backend-to-azure)
- [Register the application](https://docs.microsoft.com/azure/developer/mobile-apps/azure-mobile-apps/quickstarts/maui/#deploy-the-backend-to-azure)
```